src/api/methods.json

```diff
@@ -357,6 +357,10 @@
     "method": "get_miner_queue"
   },
   {
+    "api": "database_api",
+    "method": "get_reward_fund"
+  },
```

Comment:
The blockchain code suggest there is a string parameter "name":
https://github.com/steemit/steem/blob/9cac7d549c79f8a570e0dd38dbe301a8746eb951/libraries/app/database_api.cpp#L364
You can add `"params": ["name"]`